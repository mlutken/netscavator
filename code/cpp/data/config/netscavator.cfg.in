php-ini-path=@PHP_INI_PATH@



