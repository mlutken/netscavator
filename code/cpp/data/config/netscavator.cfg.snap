php-ini-path=/bin/php.ini


